From b6a8e670be8562b7b3c3b05d0240741359dcf667 Mon Sep 17 00:00:00 2001
From: Maria <masha.burdugova@mail.ru>
Date: Sat, 16 Mar 2024 22:08:28 +0000
Subject: [PATCH 1/2] homework

---
 20240311/0/com.py | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)
 create mode 100644 20240311/0/com.py

diff --git a/20240311/0/com.py b/20240311/0/com.py
new file mode 100644
index 0000000..5f18539
--- /dev/null
+++ b/20240311/0/com.py
@@ -0,0 +1,23 @@
+import cmd
+
+class Echoer(cmd.Cmd):
+    """Echo console"""
+    prompt = "~ "
+    words = "one", "two", "three","four", "five"
+
+    def complete_echo(self, text, line, begidx, endeidx):
+        return [c for c in self.words if c.startwith(text)]
+    
+    def do_echo(self, args):
+        """echo any string"""
+        print(args)
+
+    def do_EOE(self, args):
+        return True
+    
+    def emptyline(self):
+        pass
+
+
+if __name__ == "__main__":
+    Echoer.cmdloop()
-- 
2.44.0

